### DISCUSSION TITLE
Is there a way to just display image using path?

### DISCUSSION THREAD

**Discussion by tarunjain1st** [anuraghazra/github-readme-stats#3535] [General]
how to add images in readme using absolute/relative path after storing image to GitHub repo.
* able to use drag option which upload image to githubuserbase, but it can't be delete

**Comment by rickstaa** [comment]
@tarunjain1st This discussion is meant for questions related to GitHub Readme Stats. Neverteless, StackOverflow has your answer: https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github.

**Comment by tarunjain1st** [comment]
it's not working, image are not visible using path after uploading to github repo.
can please try once in any of readme, first upload image as file then try to display that in readme

**Reply by rickstaa** [reply]
Hey @tarunjain1st, just a quick heads up! Storing image binaries directly on GitHub isn't the best practice (check out why here: https://softwareengineering.stackexchange.com/questions/80962/should-images-be-stored-in-a-git-repository). If it's just a handful, no problem. The solution in https://stackoverflow.com/a/76781456/8135687 works on my side 🤔. Maybe [this guide](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images) helps you solve the difficulties you are having. Good luck with your issue!
